### ISSUE TITLE
Does not catch error in invalid RUN command

### ISSUE THREAD

**Issue by prog1dev** [RedCoolBeans/dockerlint#40]
### Issue details

Linter does not catch error

### Steps to reproduce/test case

Dockerfile:
```
FROM ruby:2.1.5

RUN
 apt-get update && apt-get install -y libxslt-dev libxml2-dev screen rsync
```

Here is output of `docker build` command:
```
Error response from daemon: Dockerfile parse error line 4: unknown instruction: APT-GET
```

### Affected version(s)

dockerlint version: `0.3.1`
node: `v7.10.1`
npm: `4.2.0`

**Comment by rnagy** [comment]
Please test with commit 016185a

The issue is not with the RUN command but with the next line which
is an unknown instruction.
